lib/helpers/utils.js

```diff
@@ -10711,16 +10711,18 @@ function get_python_command_from_env(env) {
   const python3_exe_name = isWin ? "python3.exe" : "python3";
   let python_cmd_to_use = PYTHON_CMD;
   if (env.VIRTUAL_ENV) {
-    if (existsSync(join(env.VIRTUAL_ENV, "bin", python_exe_name))) {
-      python_cmd_to_use = join(env.VIRTUAL_ENV, "bin", python_exe_name);
-    } else if (existsSync(join(env.VIRTUAL_ENV, "bin", python3_exe_name))) {
-      python_cmd_to_use = join(env.VIRTUAL_ENV, "bin", python3_exe_name);
+    const bin_dir = isWin ? "Scripts" : "bin";
+    if (existsSync(join(env.VIRTUAL_ENV, bin_dir, python_exe_name))) {
+      python_cmd_to_use = join(env.VIRTUAL_ENV, bin_dir, python_exe_name);
+    } else if (existsSync(join(env.VIRTUAL_ENV, bin_dir, python3_exe_name))) {
+      python_cmd_to_use = join(env.VIRTUAL_ENV, bin_dir, python3_exe_name);
     }
   } else if (env.CONDA_PREFIX) {
-    if (existsSync(join(env.CONDA_PREFIX, "bin", python_exe_name))) {
-      python_cmd_to_use = join(env.CONDA_PREFIX, "bin", python_exe_name);
-    } else if (existsSync(join(env.CONDA_PREFIX, "bin", python3_exe_name))) {
-      python_cmd_to_use = join(env.CONDA_PREFIX, "bin", python3_exe_name);
+    const bin_dir = isWin ? "" : "bin";
```

Comment:
Are we sure there is no `Scripts` for conda?

Reply:
@prabhu @aryan-rajoria I just checked. Conda also uses Scripts rather than bin.

Reply:
The script folder does not have the python executable, therefore it gives error.  I had tried to run it on my windows machine will share screenshots

Reply:
![{7A93522B-3C3E-48AE-BD35-03C58029B547}](https://github.com/user-attachments/assets/52a4e488-0804-4c82-b1db-184240f653eb)

Reply:
Have you tried printing `python_cmd_to_use`. If we use blank in join, it will add an extra slash so the full path will be incorrect. So it should be like below without the bin_dir.

```
if (existsSync(join(env.CONDA_PREFIX, python_exe_name))) {
      python_cmd_to_use = join(env.CONDA_PREFIX, python_exe_name);
    }
```